packages/scene-graph-mediator/runtime/test/importer/Pixi.js

```diff
@@ -17,6 +18,14 @@ function clearCache() {
   PIXI.BaseTexture.removeFromCache(testSpriteUrl);
 }
 
+function sleep(msec) {
```

Comment:
it seems unused.

Reply:
deleted unused methods.
https://github.com/drecom/scene-graph/pull/28/commits/531faf869cbe5c519477af9d9482ec60713fb46e